### PR TITLE
Seed Project Path Update

### PR DESCRIPTION
**Pull request by mehreencs87**
### Your checklist for this pull request

Please review the [guidelines for contributing](https://github.com/auth0/docs#contributing) to this repository.
- [ ] Content conforms to our [Contributing Guidelines](https://github.com/auth0/docs#contributing-guidelines)
- [ ] If applicable, you have added details to the [update feed](https://github.com/auth0/docs/tree/master/updates) 

If you are an Auth0 employee, after you submit this PR create and test a [Review App](https://github.com/auth0/docs#review-apps) to test your changes.
### Description

Please describe your pull request.

Thank you!

### CODE REVIEW

File: articles/client-platforms/ember2js/01-login.md [hunk 1]
```diff
@@ -17,10 +17,10 @@ This tutorial and seed project have been tested with the following:
 :::
 
 <%= include('../../_includes/_package', {
-  pkgRepo: 'auth0-ember-simple-auth',
```
Comment (mpaktiti) [inline articles/client-platforms/ember2js/01-login.md:20]: @mehreencs87 Please add the following line:
`githubUrl: 'https://github.com/auth0-samples/auth0-emberjs2-sample/tree/master/00-Starter-Seed'`

See the updated [contributing guidelines](https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#seed-projects) for details.

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

**Comment by mehreencs87** [comment]
Done . Check the latest commit. @mpaktiti

File: articles/client-platforms/ember2js/01-login.md (hunk 1 above)
Comment (mpaktiti) [inline articles/client-platforms/ember2js/01-login.md:20]: @mehreencs87 Hi, another change was introduced and we need to apply it before this PR is merged. `pkgOrg` was added:
- if the sample is under `auth0-samples` repo add the line: `pkgOrg: 'auth0-samples'`
- if the sample is under `auth0` repo add the line: `pkgOrg: 'auth0'`
  Details: https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#seed-projects

Can you please update this and rebase this PR? Thanks.

**Comment by mehreencs87** [comment]
@mpaktiti updated, check the latest commit.

**Comment by mehreencs87** [comment]
@mpaktiti updated fix here https://github.com/auth0/docs/pull/2032

**Pull request closed without merging**